### PR TITLE
[script.musicfinder] 1.0.0

### PR DESCRIPTION
**Pull request by Razzeee**
Pull request for https://github.com/xbmc/repo-scripts/issues/453

Please check thorough. Code was not done by me, I'm just doing the PR.

### CODE REVIEW

File: script.musicfinder/addon.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<addon id="script.musicfinder" name="Music Finder Script" provider-name="theDeadMan" version="1.0.0">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.simplejson" version="2.0.10"/>
```
Comment (enen92) [inline script.musicfinder/addon.xml]: no need for this I think

File: script.musicfinder/default.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#    This script is a modification of service.skin.widgets and service.library.data.provider
+#    Thanks to the original authors: BigNoid, Unfledged and Martijn.
+
+import os
+import sys
+import xbmc
+import xbmcgui
+import xbmcaddon
+
+try:
```
Comment (enen92) [inline script.musicfinder/default.py]: same

File: script.musicfinder/addon.xml (hunk 1 above)
Comment (Razzeee) [inline script.musicfinder/addon.xml]: @chrisbevan please change this for the next version, shouldn't hurt for now

**Comment by Razzeee** [comment]
If those are the only problems, please let's pull as it is and improve with the next versions

File: script.musicfinder/addon.xml (hunk 1 above)
Comment (chrisbevan) [inline script.musicfinder/addon.xml]: Thanks, will do.

File: script.musicfinder/default.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    import simplejson as json
+except Exception:
+    import json
+
+__addon__        = xbmcaddon.Addon()
```
Comment (ronie) [inline script.musicfinder/default.py]: we're also trying to get rid of the `__name__` formatting.

please use `addon = ` or `ADDON = ` in the next update

**Pull request merged**